desktop-app/src/renderer/components/Button/index.tsx

```diff
@@ -48,8 +48,8 @@ const Button = ({
     hoverBg = 'hover:bg-slate-200';
     hoverBgDark = 'dark:hover:bg-slate-700';
   } else if (isPrimary) {
-    hoverBg = 'hover:bg-emerald-600';
-    hoverBgDark = 'dark:hover:bg-emerald-600';
+    hoverBg = 'hover:bg-slate-200';
```

Comment:
I guess, we need to give them more emphasis if they're `primary` as compared to the colors above?

Reply:
I standardized the button color to slate for consistency since both buttons served the same function. This ensures a unified user experience.

Reply:
The thing is the `Primary` variant is an emphasised version and so it is designed to have the brand colour bg.